web_app/tests/test_routes.py

```diff
@@ -0,0 +1,47 @@
+import os
+import sys
+
+sys.path.append(os.path.dirname(os.path.dirname(__file__)))
+from app import app
+
+
```

Comment:
```suggestion
from web_app.app import app

```

Unnecessary to append web_app.app path – the project root path is already in Python's path